src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -1471,3 +1484,27 @@ def get_pipeline_nodes(self):
     def get_data(self, *args, **kwargs):
         assert len(self.data) > 0, f"You must run the extension {self.extension_name} before retrieving data"
         return self._get_data(*args, **kwargs)
+
+
+# this is a hardcoded list to to improve error message and autoload mechanism
+# this is important because extension are register when the submodule is imported
```

Comment:
```suggestion
# this is important because extension are registered when the submodule is imported
```

Reply:
We should get a spell check action. We are spending a lot of energy on these type of errors.

Reply:
Haha, I would love to hear the convo that convinces Sam to add another linter/formatter.

Reply:
I am not thinking on a linter or formater, there are CI actions that just point out the spell mistakes. We can just check them there instead of fishing for them and relying on our nose.